javav2/example_code/lookoutvision/src/main/java/com/example/lookoutvision/UpdateDatasetEntries.java

```diff
@@ -0,0 +1,81 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+// snippet-start:[lookoutvision.java2.update_dataset_entries.complete]
+
+package com.example.lookoutvision;
+
+import software.amazon.awssdk.services.lookoutvision.LookoutVisionClient;
+import software.amazon.awssdk.services.lookoutvision.model.DatasetStatus;
+import software.amazon.awssdk.services.lookoutvision.model.LookoutVisionException;
+
+import java.io.FileNotFoundException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+// Updates dataset entries in an Amazon Lookout for Vision dataset.
+public class UpdateDatasetEntries {
+
+    public static final Logger logger = Logger.getLogger(UpdateDatasetEntries.class.getName());
+
+    public static void main(String[] args) throws Exception {
+
+        String projectName = null;
+        String datasetType = null;
+        String updateFile = null;
+
+        final String USAGE = "\n" +
+                "Usage:\n" +
+                "    UpdateDatasetEntries <project_name> <dataset_type> <manifest_file> \n\n" +
+                "Where:\n" +
+                "    project_name - the name of the project that you want to use.\n" +
+                "    dataset_type - the type (train or test) of the dataset that you want to update.\n" +
```

Comment:
"    dataset_type - The type of the dataset that you want to update (train or test).\n" +

Reply:
Fixed.